keybase/report.go

```diff
@@ -52,6 +52,8 @@ func (c context) reportAction(action updater.UpdateAction, options updater.Updat
 	data.Add("version", options.Version)
 	data.Add("upd_version", options.UpdaterVersion)
 	data.Add("action", action.String())
+	autoUpdate, _ := c.config.GetUpdateAuto()
```

Comment:
If we've made the change on this iteration through the loop, is it reflected in this getter?